tests/Database/DatabaseFunctionalTest.php

```diff
@@ -152,6 +153,23 @@ public function testGetSelectsCollectionAndInheritsOptions()
         $this->assertSame(WriteConcern::MAJORITY, $debug['writeConcern']->getW());
     }
 
+    public function testModifyCollection()
+    {
+        $this->database->createCollection($this->getCollectionName());
+
+        $indexes = [['key' => ['lastAccess' => 1], 'expireAfterSeconds' => 3]];
+        $createIndexes = new CreateIndexes($this->getDatabaseName(), $this->getCollectionName(), $indexes);
+        $createIndexes->execute($this->getPrimaryServer());
+
+        $keyPattern = (object) ['lastAccess' => 1];
```

Comment:
Is the `(object)` cast necessary? `['lastAccess' => 1]` should already serialize as a BSON document.

If it's not needed, can we just use a `['lastAccess' => 1]` literal below and skip this variable?